src/RoveCommLoggers.h

```diff
@@ -21,8 +21,8 @@
 #include <iostream>
```

Comment:
Put implicit include in 
/// \cond
#include <array>

/// \endcond
to avoid them being included in doxygen graph.